tests/draft2019-09/optional/format/time.json

```diff
@@ -5,9 +5,64 @@
         "tests": [
             {
                 "description": "a valid time string",
+                "data": "08:30:06Z",
+                "valid": true
+            },
+            {
+                "description": "a valid time string with leap second",
+                "data": "00:00:60Z",
+                "valid": true
+            },
+            {
+                "description": "a valid time string with second fraction",
                 "data": "08:30:06.283185Z",
                 "valid": true
             },
+            {
+                "description": "a valid time string with plus offset",
+                "data": "08:30:06+00:20",
+                "valid": true
+            },
+            {
+                "description": "a valid time string with minus offset",
+                "data": "08:30:06-08:00",
+                "valid": true
+            },
+            {
+                "description": "a valid time string with case-insensitive Z",
+                "data": "08:30:06z",
```

Comment:
Are you sure this is valid? There is no indication that the Z is case-insensitive.

Reply:
There's this note below ABNF in the spec, so I think so:

> NOTE: Per [ABNF] and ISO8601, the "T" and "Z" characters in this syntax may alternatively be lower case "t" or "z" respectively.

There is a test for lower case in `date-time.json`:

```json
{
  "description": "case-insensitive T and Z",
  "data": "1963-06-19t08:30:06.283185z",
  "valid": true
}
```

Reply:
yes, I edited my comment earlier :)